projects/admin/src/app/service/record-routing.service.ts

```diff
@@ -50,6 +50,10 @@ import { PatronDetailViewComponent } from '../record/detail-view/patron-detail-v
 import { DocumentEditorComponent } from '../record/custom-editor/document-editor/document-editor.component';
 import { VendorDetailViewComponent } from '../record/detail-view/vendor-detail-view/vendor-detail-view.component';
 import { VendorBriefViewComponent } from '../record/brief-view/vendor-brief-view.component';
+import { AcqOrderBriefViewComponent } from '../record/brief-view/acq-order-brief-view.component';
+import { AcqOrderDetailViewComponent } from '../record/detail-view/acq-order-detail-view/acq-order-detail-view.component';
+import { AcqOrderLineDetailViewComponent } from '../record/detail-view/acq-order-line-detail-view/acq-order-line-detail-view.component';
+import { formatDate } from '@angular/common';
```

Comment:
imports are not sorted correctly, you can use "Organize imports" command, for instance

Reply:
ok